graphene/src/layers/text/to_kurbo.rs

```diff
@@ -74,7 +74,7 @@ pub fn to_kurbo(str: &str, buzz_face: rustybuzz::Face, font_size: f64, line_widt
 		path: BezPath::new(),
 		pos: Point::ZERO,
 		offset: Vec2::ZERO,
-		ascender: buzz_face.ascender() as f64,
+		ascender: (buzz_face.ascender() as f64 / buzz_face.height() as f64) * font_size / scale,
```

Comment:
Descenders below the bounding box is to be expected. This change seems to add a large basically a full line below the text for the bounding box, which isn't ideal.

Reply:
> This change seems to add a large basically a full line below the text for the bounding box, which isn't ideal.

No this does not. I am not sure what you are talking about. This does not influence the bounding box at all.

Reply:
I've reverted the baseline change, however the new line issue probably still remains.

Reply:
I figured out the problem. Multiple lines cause there to be a large gap underneath the text, but this is also the case in master. I thought that had to do with this baseline change which is where I first noticed the issue. But it actually had nothing to do with it.

![](https://files.keavon.com/-/DearestGlaringBaleenwhale/capture.png)

Sorry for the confusion.

Reply:
@Keavon No problem, it appears that chrome returns a random trailing `\n` char in some situations with `textContent`. Converting to `innerText` appears to have solved this issue?

Reply:
Awesome, looks good. The only text issues I'm seeing now is that we seem to have *nothing*(?) highlighted when you first click to create a new text field and this shows up as a block of blue (you can zoom into it with Ctrl+scroll). Then typing something and deleting it again makes it so that goes away. Also it seems that zooming into the document in general causes all the text to get selected. But both of these can be eventually resolved when we have more time to polish, they aren't important enough for current priorities.

Reply:
> Awesome, looks good. The only text issues I'm seeing now is that we seem to have nothing(?) highlighted when you first click to create a new text field and this shows up as a block of blue

Resolved.